### DISCUSSION TITLE
image links are not working

### DISCUSSION THREAD

**Discussion by yana-khusainova** [thuliteio/doks#610] [Q&A]
Hi!

I am making link on a image these ways:
1. `<img  src = "../5.jpg" alt="5">` 
2. `<img  src = "5.jpg" alt="5">` 
3. `![5](5.jpg)`
and no one works:
![image](https://user-images.githubusercontent.com/88020207/149318991-af9bbaa6-6983-45e9-a292-f4577af6a854.png)

image and md file in one directory. What can I do in this case? 

Thank you!

P.S. i added code from this https://github.com/h-enk/doks/issues/535 to render.image

**Comment by h-enk** [comment]
> image and md file in one directory. 

Like so?

```bash
..
├── blog/
│   ├── say-hello-to-doks/
│   │   ├── index.md
│   │   └── say-hello-to-doks.png
│   └── _index.md
└── _index.md
```

Then, in `./blog/say-hello-to-doks/index.md`, this should work:

```
![Say hello to Doks](say-hello-to-doks.png)
```

Would you mind sharing your repo?

**Reply by yana-khusainova** [reply]
Unfortuantely I can't share the repo because of privacy policy of my company. 

i have structure like this:
├── docs/
│   ├── introduction/
│   │   ├── index.md
│   │   └── 5.jpg
│   │   ├── faq.md    
│   └── _index.md
└── _index.md

and I need an image 5 (5.jpg) in `./docs/introduction/faq.md`

**Reply by h-enk** [reply]
Set up your structure like so:

```bash
..
├── docs/
│   ├── introduction/
│   │   ├── index.md
│   │   └── faq/
│   │     ├── 5.jpg
│   │     └── index.md
│   └── _index.md
└── _index.md
``` 

See also the Hugo docs: [Page Bundles](https://gohugo.io/content-management/page-bundles/)

**Reply by yana-khusainova** [reply]
oh got it! thank you very much!! it was really helpful :)

**Reply by h-enk** [reply]
You're welcome!
